SCALESET_README.md

```diff
@@ -0,0 +1,39 @@
+## Create Cluster
+
+`kind create cluster`
+
+## Install certificate manager
+`kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml`
+
+## Export GITHUB_TOKEN
+
+Create a GitHub PAT with the [appropriate permissions](https://github.com/thboop/actions-runner-controller#deploying-using-github-app-authentication)
+
+Make sure the PAT has the right permissions depending on your use case (repo only, admin of org or enterprise)
+
+`export GITHUB_TOKEN=ghp_13xe9hrjhNm2eLN23r5V6OFuj4qAD3nndozQ`
```

Comment:
replace the real token with something else? also make sure the revoke the token.

Reply:
The token is false and human-generated :)